cmd/server/controllers/EmployeeController.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/vinigracindo/mercado-fresco-stranger-strings/internal/domains/employees"
+)
+
+type EmployeeController struct {
+	service employees.Service
+}
+
+func NewEmployee(service employees.Service) EmployeeController {
+	return EmployeeController{service: service}
+}
+
+func (controller EmployeeController) GetAll() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		employees, err := controller.service.GetAll()
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{
```

Comment:
Dúvida, aqui vem o erro que foi tratado lá no repository, certo?
Se for, podemos retornar um StatusNotFound ao invés de StatusInternalServerError?